pysatModels/models/pydineof_dineof.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Supports exported model data from pyDINEOF, a Python package
+Support exported model data from pyDINEOF, a Python package
 that interfaces with a version of Data Interpolation Empirical
 Orthogonal Functions (DINEOFs). This module couples into
 the systematic export pyDINEOF format and thus should support
```

Comment:
This needs to be reduced to one line, the rest can be moved to the "Note" section.